components/org.wso2.carbon.identity.organization.management.service/src/main/java/org/wso2/carbon/identity/organization/management/service/constant/OrganizationManagementConstants.java

```diff
@@ -93,6 +93,8 @@ public class OrganizationManagementConstants {
     public static final String ORGANIZATION_MGT_CONFIG_FILE = "organization-mgt.xml";
     public static final String IS_CARBON_ROLE_VALIDATION_ENABLED_FOR_LEVEL_ONE_ORGS =
             "LevelOneOrganizationConfigs.EnableCarbonRoleBasedValidation";
+    public static final String IS_ORG_QUALIFIED_URLS_SUPPORTED_FOR_LEVEL_ONE_ORGS =
+            "LevelOneOrganizationConfigs.SupportOrganizationQualifiedURLs";
```

Comment:
```suggestion
            "LevelOneOrganizationConfigs.DisableOrganizationQualifiedURLs";
```

Reply:
The config has changed https://github.com/wso2-enterprise/identity-org-mgt-connector/pull/168 and resolving the comment